api_event.go

```diff
@@ -60,10 +60,6 @@ func (s *service) GroupMetadataList(req *protocoltypes.GroupMetadataList_Request
 		}
 		defer sub.Close()
 		newEvents = sub.Out()
-	} else {
-		noop := make(chan interface{})
-		newEvents = noop
```

Comment:
If you don't assign newEvents, then it will be nil when it is used here:
https://github.com/berty/weshnet/blob/2dcad185271142011529e32a07be44f1de5ec3a5/api_event.go#L105

Is that a problem?

Reply:
That's the whole point, if not Subscribing to new metadata events, newEvents should be `nil` and block indefinitely:
see this exemple: https://go.dev/play/p/wgtLJyq9-M_5

Reply:
OK.